tiers/migrations/0002_auto_20170321_1856.py

```diff
@@ -21,6 +21,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='tier',
             name='organization',
-            field=models.OneToOneField(related_name='tier', null=True, blank=True, to=ORGANIZATION_MODEL),
+            field=models.OneToOneField(related_name='tier', null=True, blank=True, to=ORGANIZATION_MODEL, on_delete=models.deletion.DO_NOTHING),
```

Comment:
Now I think that's not ideal but okay to have. `CASCADE` should have been used otherwise it will be `organization_id` would be left null and orphaned in the database.